app/src/main/java/com/example/lamnn/todo_mvp/data/repository/TasksContent.java

```diff
@@ -0,0 +1,43 @@
+package com.example.lamnn.todo_mvp.data.repository;
+
+import android.os.AsyncTask;
+
+import com.example.lamnn.todo_mvp.data.model.Task;
+import com.example.lamnn.todo_mvp.data.resource.Callback;
+
+import org.json.JSONException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TasksContent extends AsyncTask<String, Void, List<Task>> {
+    private Callback<List<Task>> mCallback;
+    private Exception mException;
+
+    public TasksContent(Callback<List<Task>> mCallback) {
```

Comment:
https://viblo.asia/p/mot-so-tips-tricks-khi-lam-viec-voi-android-studio-yMnKMJjgZ7P
tip trick số 6.
Sau khi làm xong tip trick số 6 e xoá constructor đi gen lại sẽ ra kết quả 
```suggestion
    public TasksContent(Callback<List<Task>> callback) {
```